docs/ecctl_deployment_create.adoc

```diff
@@ -6,38 +6,90 @@ Creates a deployment
 [float]
 === Synopsis
 
-Creates a deployment which is defined from a file definition using the --file=+++<file path="">+++(shorthand: -f) flag.+++</file>+++
+Creates a deployment which can be defined through flags or from a file definition.
+Sane default values are provided, making the command work out of the box even when no parameters are set.
+When version is not specified, the latest available stack version will automatically be used.
+These are the available options:
 
-You can create a definition by using the sample JSON:
-  https://elastic.co/guide/en/cloud/current/ec-api-deployment-crud.html#ec_create_a_deployment
+* Simplified flags to set size and zone count for each instance type (Elasticsearch, Kibana, APM and App Search).
```

Comment:
I can't comment on line :27 of `create_help.go`, but I think we need to add the Enterprise Search too, here.